macros/src/main/scala-2/com/softwaremill/macwire/internals/ConstructorCrimper.scala

```diff
@@ -44,22 +44,22 @@ private[macwire] class ConstructorCrimper[C <: blackbox.Context, T: C#WeakTypeTa
 
   lazy val constructorParamLists: Option[List[List[Symbol]]] = constructor.map(_.asMethod.paramLists.filterNot(_.headOption.exists(_.isImplicit)))
 
-  lazy val constructorArgs: Option[List[List[Tree]]] = log.withBlock("Looking for targetConstructor arguments") {
```

Comment:
same here, maybe we can introduce a second arg list to `ConstructorCrimper` and move the dependency there